docs/tutorials/build-a-module.rst

```diff
@@ -5,5 +5,338 @@
+and loads the default configuration from ``config.py``. 
+
+version.py
+>>>>>>>>>>
+File containing the version of your module.
+
+views.py
+>>>>>>>>
+Here you declare the views or endpoints you want to expose. By default, it creates a
+simple view on the root end point that renders a template.
+
+templates
+>>>>>>>>>
+All your Jinja templates should be stored in this folder. A Jinja template is an HTML file that can be modified according to some parameters.
+
+static
+>>>>>>
+If your module contains JavaScript or CSS files, they should go in a folder called ``static``. Also, if you want to group them in bundles,
+you should add a ``bundles.py`` file next to the ``static`` folder.
+
+
+Install a module
+----------------
+
+First of all, create a virtualenv for the module:
+
+.. code-block:: console
+
+    $ mkvirtualenv my_venv
+
+Installing the module is very easy, you just need to go to its root directory
+and `pip install` it:
+
+.. code-block:: console
+
+    (my_venv)$ cd invenio-foo/
+    (my_venv)$ pip install --editable .[all]
+
+Some explanations about the command:
+
+- The ``--editable`` option is used for development. It means that if you change the
+  files in the module, you won't have to reinstall it to see the changes. In a
+  production environment, this option shouldn't be used.
+- The ``.`` is in fact the path to your module. As we are in the root folder of
+  the module, we can just say *here*, which is what the dot means.
+- The ``[all]`` after the dot means we want to install all dependencies, which
+  is common when developing. Depending on your use of the module, you can
+  install only parts of it:
+
+    - The default (nothing after the dot) installs the minimum to make the
+      module run.
+    - ``[tests]`` installs the requirements to test the module.
+    - ``[docs]`` installs the requirements to build the documentation.
+    - Some modules have extra options.
+
+If you need multiple options, you can chain them: ``[tests,docs]``.
+
+
+.. _run-the-tests:
+
+Run the tests
+-------------
+In order to run the tests, you need to have a valid git repository. The
+following steps need to be run only once. Go into the root folder of the module:
+
+.. code-block:: console
+
+    (my_venv)$ git init
+    (my_venv)$ git add --all
+    (my_venv)$ check-manifest --update
+
+What we have done:
+
+- Change the folder into a git repository, so it can record the changes made to
+  the files.
+- Add all the files to this repository.
+- Update the file ``MANIFEST.in`` (this file controls which files are included
+  in your Python package when it is created and installed).
+
+Now, we are able to run the tests:
+
+.. code-block:: console
+
+    (my_venv)$ ./run-tests.sh
+
+.. _run-the-example-app:
+
+Build the documentation
+-----------------------
+The documentation can be built with the ``run-tests.sh`` script, but you need
+to have installed the package with its *tests* requirements. If you just want to build the
```

Comment:
I think `but you need to have installed the package` could be replaced by `but you need to have the package installed`